### PR TITLE
Update virtual-list support DOM7

### PR DESCRIPTION
**Pull request by jinzhe**

### CODE REVIEW

**Comment by nolimits4web** [comment]
What do you mean by "support Dom7"? It is already supported. If in your example, you will pas HTMLElement as listBlock parameter nothing will work at all

**Comment by ZanderBrown** [comment]
Along with a description this pull request is broken according to travis. Please fix this.

File: src/js/virtual-list.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ var VirtualList = function (listBlock, params) {
 
     // Preparation
     var vl = this;
-    vl.listBlock = $(listBlock);
+    vl.listBlock = typeof listBlock=="string"?$(listBlock):listBlock;
     vl.params = params;
```
Comment (ZanderBrown) [inline src/js/virtual-list.js]: What is this doing and why?

You mentioned DOM7 but this isn't part of the DOM7 library

**Comment by ZanderBrown** [comment]
Unexplained, has conflicts and no response has been given.

**Pull request closed without merging**